x-pack/plugins/timelines/public/components/t_grid/toolbar/bulk_actions/alert_status_bulk_actions.tsx

```diff
@@ -13,29 +13,32 @@ import type {
   SetEventsDeleted,
   OnUpdateAlertStatusSuccess,
   OnUpdateAlertStatusError,
+  CustomBulkActionProp,
 } from '../../../../../common/types';
 import type { Refetch } from '../../../../store/t_grid/inputs';
 import { tGridActions, TGridModel, tGridSelectors, TimelineState } from '../../../../store/t_grid';
 import { BulkActions } from '.';
-import { useStatusBulkActionItems } from '../../../../hooks/use_status_bulk_action_items';
+import { useBulkActionItems } from '../../../../hooks/use_status_bulk_action_items';
```

Comment:
@academo I suggest you rename this file `use_status_bulk_action_items` to `use_bulk_action_items`